src/guides/v2.4/release-notes/release-notes-2-4-0-open-source.md

```diff
@@ -1532,20 +1561,20 @@ We have fixed hundreds of issues in the Magento 2.4.0 core code.
 
 <!--- MC-25176-->
 
-*  Magento no longer throws an exception when a customer uses the storefront quick search in multi-website deployments.
-
-<!--- ENGCOM-6364-->
-
-*  The Elasticsearch versions are now identified by X.x notation instead of X.0+ notation in the Admin (**Stores** > **Settings** > **Configuration** > **Catalog** > **Catalog** > **Catalog Search**). _Fix submitted by Andreas Mautz in pull request [25838](https://github.com/magento/magento2/pull/25838)_. [GitHub-25674](https://github.com/magento/magento2/issues/25674)
+*  Magento no longer throws an exception when a customer uses the storefront  quick search in multi-website deployments.
 
 <!--- ENGCOM-6842-->
 
 *  Magento no longer displays a warning when a merchant opens the search term page by clicking on the Search terms option on the footer of **Marketing** > **SEO & Search** > **Search Terms**.  _Fix submitted by vishal-webkul in pull request [25246](https://github.com/magento/magento2/pull/25246)_. [GitHub-25245](https://github.com/magento/magento2/issues/25245)
 
-<!-- ENGCOM-7289-->
+<!--- ENGCOM-7289-->
 
 *  Magento no longer resizes the viewport when you click on the search input box when running Safari in mobile view.  _Fix submitted by Pawe Tylek in pull request [27603](https://github.com/magento/magento2/pull/27603). [GitHub-27506](https://github.com/magento/magento2/issues/27506)_
 
+<!--- ENGCOM-6364-->
+
+*  The Elasticsearch versions are now identified by X.x notation instead of X.0+ notation in the Admin (**Stores** > **Settings** > **Configuration** > **Catalog** > **Catalog** > **Catalog Search**). _Fix submitted by Andreas Mautz in pull request [25838](https://github.com/magento/magento2/pull/25838)_. [GitHub-25674](https://github.com/magento/magento2/issues/25674)
```

Comment:
```suggestion
*  The Elasticsearch versions are now identified by X.x notation instead of X.0+ notation in the Admin (**Stores** > Settings > **Configuration** > **Catalog** > **Catalog** > **Catalog Search**). _Fix submitted by Andreas Mautz in pull request [25838](https://github.com/magento/magento2/pull/25838)_. [GitHub-25674](https://github.com/magento/magento2/issues/25674)
```

Reply:
Changed in both files and searched for other occurrences